sql/mozfun/norm/truncate_version/udf.sql

```diff
@@ -0,0 +1,10 @@
+CREATE OR REPLACE FUNCTION norm.truncate_version(os_version STRING)
+RETURNS STRING AS (
+  REGEXP_REPLACE(os_version, r"^([0-9]+.[0-9]+).*", "\\1")
+);
+
+-- Tests
+SELECT
+  assert.equals("16.1", norm.truncate_version("16.1.1")),
+  assert.equals("10", norm.truncate_version("10")),
+  assert.equals("5.1", norm.truncate_version("5.1.0-ubuntu-foobar"))
```

Comment:
Can you add one more test case that tests something that doesn't fit the version pattern at all, like `foo-bar`, or `10-a`? I expect it to return null.